src/pages/Information/Information.jsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 const Information = () => (
   <>
     <div data-test="information-display">
-      <h2 data-test="heading-display">What is HacktoberFest?</h2>
+      <h2 data-test="heading-display">What is Hacktoberfest</h2>
```

Comment:
:question: Did you mean to remove the question mark?

Reply:
Addressed:  a379eb7b3721a74ff9e1e84d867dfc18c86eb75b